<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_ProgramUpgrade</class>
 <widget class="QWidget" name="Form_ProgramUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form_ProgramUpgrade</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form_ProgramUpgrade{
background-image: url(:/pic_de260/bg_01.png);
}

QTableView{
	color: rgb(196, 160, 0);
	font: 17pt &quot;WenQuanYi Zen Hei&quot;;
	background-color: transparent;
	gridline-color:rgba(73,153,216,200);
	selection-background-color:rgb(170,255,255);
	selection-color:rgb(255,0,0);
}

QLabel{
color: white;
font: 15pt &quot;WenQuanYi Zen Hei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>133</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>U盘状态:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>133</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>173</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>升级包:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="wika_tableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>210</y>
     <width>612</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section:first{
	color:rgb(73,153,216);
	background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0,103,185, 255), stop:1 rgba(0,0,0, 255));
   height:30px;
	font: bold 17pt &quot;WenQuanYi Zen Hei&quot;; 
   border:1px solid rgba(73,153,216,200);
}

QHeaderView::section:middle{
	color:rgb(73,153,216);
	background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0,103,185, 255), stop:1 rgba(0,0,0, 255));
   height:30px;
	font: bold 17pt &quot;WenQuanYi Zen Hei&quot;; 
   border:1px solid rgba(73,153,216,200);
}

QHeaderView::section:last{
	color:rgb(73,153,216);
	background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0,103,185, 255), stop:1 rgba(0,0,0, 255));
   height:30px;
	font: bold 17pt &quot;WenQuanYi Zen Hei&quot;; 
   border:1px solid rgba(73,153,216,200);
}

/*
QHeaderView::section:checked{
	background-color:red;
}

QTableView::item{
    min-height: 30px;
}
*/
background-color: rgb(239, 41, 41);</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>176</y>
     <width>612</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>程序升级</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>133</y>
     <width>241</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(204, 0, 0);</string>
   </property>
   <property name="text">
    <string>请检查文件选择是否正确!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1808</x>
     <y>20</y>
     <width>110</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/button/button5.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>50</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 17pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>中　断</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>wika_tableView</class>
   <extends>QTableView</extends>
   <header location="global">wika_tableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
